Jenkinsfile

```
@@ -88,6 +88,14 @@ pipeline {
             }
         }
     }
+    post { 
+        always { 
```

Maybe is better do the clean in the *cleanup* and leave the *always* for other post actions

```suggestion
        cleanup{ 
```